src/software/ai/hl/stp/tactic/get_behind_ball_tactic.cpp

```diff
@@ -0,0 +1,65 @@
+#include "software/ai/hl/stp/tactic/get_behind_ball_tactic.h"
+
+#include <algorithm>
+
+GetBehindBallTactic::GetBehindBallTactic(bool loop_forever)
```

Comment:
are get_behind_ball_tactic and get_behind_ball_fsm basically duplicates using different frameworks?